cpp/src/parquet/column_writer_test.cc

```diff
@@ -645,6 +645,32 @@ TEST(TestWriter, NullValuesBuffer) {
                                  valid_bits_offset, values);
 }
 
+TEST(TestWriter, EnableChecksum) {
```

Comment:
This is a bit too minimal IMHO. We should have roundtrip tests for both v1 and v2 data pages, to make sure that the writer generates checksums as expected by the reader.

(ideally there's also a way to check that checksums are really written out...)

Reply:
I think you're right, let me add a roundtrip test later.

How can we check checksums are written out? Seems there is no efficent way here. If I miss something, please tell me

Reply:
Unfortunately not indeed. It would be nice if the Parquet writer could emit some statistics, but that's too large a change for this PR.